mathics/builtin/colors/color_directives.py

```diff
@@ -401,15 +404,15 @@ def apply(self, c1, c2, evaluation, options):
 
             def compute(a, b):
                 return Expression(
-                    "Apply",
+                    SymbolApply,
                     distance_function,
                     Expression(
-                        "List",
+                        SymbolList,
                         Expression(
-                            "List", *[Real(val) for val in a.to_color_space("LAB")]
+                            SymbolList, *[Real(val) for val in a.to_color_space("LAB")]
```

Comment:
The idom `*[conversionfn(val) for val in collection]` should encapsulated in some sort of List constructor. Probably not _the_ object constructor which, like Expression should assume everything is converted, but more like the way it is in `to_expression()`.

Reply:
something like `to_mathics_list`?

Reply:
Yes, something like `to_mathics_list()`